app/models/member.rb

```diff
@@ -157,6 +157,11 @@ def party_name
     party_object.long_name
   end
 
+  # Long version of party name ukr_version
+  def display_party_name_ukr
```

Comment:
Yes, need to create a second helper)))
I create the necessary changes or you do ?